src/cartridges/int_adyen_overlay/cartridge/scripts/adyenCheckout.js

```diff
@@ -198,18 +217,15 @@ function doPaymentsCall(order, paymentInstrument, paymentRequest) {
 
       paymentResponse.token3ds2 = responseObject.action.token;
       paymentResponse.paymentData = responseObject.paymentData;
-    } else if (
-      paymentResponse.resultCode === 'Authorised' ||
-      paymentResponse.resultCode === 'RedirectShopper'
-    ) {
+    } else if (acceptedResultCodes.indexOf(resultCode) !== -1) {
       paymentResponse.decision = 'ACCEPT';
       // if 3D Secure is used, the statuses will be updated later
-      if (paymentResponse.resultCode === 'Authorised') {
+      if (resultCode === constants.RESULTCODES.AUTHORISED) {
         order.setPaymentStatus(Order.PAYMENT_STATUS_PAID);
         order.setExportStatus(Order.EXPORT_STATUS_READY);
         Logger.getLogger('Adyen').info('Payment result: Authorised');
       }
-    } else if (paymentResponse.resultCode === 'PresentToShopper') {
+    } else if (presentToShopperResultCodes.indexOf(resultCode) !== -1) {
```

Comment:
Why creating an array for this as this is just one resultCode?

Reply:
for consistency's sake